client/src/components/classesPage/forms/CampaignForm.js

```diff
@@ -221,12 +246,41 @@ function CampaignForm(props) {
             defaultValue={today}
             onChange={e => alterTime(e)}
           />
+          <div className={classes.scheduleDiv}>
```

Comment:
If you wanted to convert the whole thing to material-ui, pretty sure you can use `Grid` for divs, looks really good w/o too though.

Reply:
Will try that.